completions/fastfetch.fish

```diff
@@ -98,9 +106,11 @@ def main():
             else:
                 print(f"{command_prefix} -f")
 
+
 if __name__ == "__main__":
     try:
         main()
-    except:
+    except Exception as e:
+        print(f"Error: {e}", file=sys.stderr)
```

Comment:
Please don't add this. This script will be run when users are typing, any unexpected logs will ruin users' experience.